modules/custom_terraform/aws-custom-terraform.md

```diff
@@ -138,6 +138,32 @@ When done, you can destroy the custom terraform resource:
 opta destroy -c custom-tf.yaml
 ```
 
+## Different options for source
```

Comment:
looking back at it now, may be we could rename the titles to reflect what the sections covers
- Example/Demo -> Use local terraform files
- Different options for source -> Use a remote terraform module
(same comment for GCP)

Reply:
done